src/SoulMemory/DarkSouls1/KnownFlag.cs

```diff
@@ -0,0 +1,182 @@
+// This file is part of the SoulSplitter distribution (https://github.com/FrankvdStam/SoulSplitter).
+// Copyright (c) 2022 Frank van der Stam.
+// https://github.com/FrankvdStam/SoulSplitter/blob/main/LICENSE
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+using System.Xml.Serialization;
+
+namespace SoulMemory.DarkSouls1
+{
+    [XmlType(Namespace = "SoulMemory.DarkSouls1")]
+    public enum KnownFlag : uint
+    {
+
+      [AnnotationAttribute(Name = "Bell of Awakening, Undead Parish 11010700", Description = "Bells of Awakening")]
```

Comment:
There is 1 added space for indentation now, but it should be 4 extra. Look at the other enums in the codebase to see.